tests/Query/Functions/Sqlite/JsonArrayTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Scienta\DoctrineJsonFunctions\Tests\Query\Functions\Sqlite;
+
+use Scienta\DoctrineJsonFunctions\Tests\Query\MysqlTestCase;
```

Comment:
This should use the `SqliteTestCase`
Extend it here: https://github.com/ScientaNL/DoctrineJsonFunctions/pull/53/files#diff-41c4b2825aaa60b9f1af7267cf128ef4R8